### PR TITLE
[NUI] Add window auixiliary message event

### PR DESCRIPTION
**Pull request by sparrow74**
### Description of Change ###
Auxiliary message is sent by display server.
When client application added the window's auxiliary hint and if the auxiliary is changed, display server send the auxiliary message.
Auxiliary message has the key, value and options.

### CODE REVIEW

File: src/Tizen.NUI/src/public/Window/WindowEvent.cs [hunk 1]
```diff
@@ -1082,5 +1083,108 @@ public void VisibiltyChangedSignalEmit(bool visibility)
...
+
+            List<string> tmpList = new List<string>();
+
+            for (int i = 0; i < size; i++)
+            {
+                string option = "none";
+                tmp.GetElementAt((uint)i).Get(out option);
+                tmpList.Add(option);
+            }
+
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/public/Window/WindowEvent.cs:1134]: please call `tmp.Dispose();` here to remove this warning =>

**src/Tizen.NUI/src/public/Window/WindowEvent.cs#L1131
Call System.IDisposable.Dispose on object created by 'tmp.GetElementAt((uint)i)' before all references to it are out of scope**

File: src/Tizen.NUI/src/public/Window/WindowEvent.cs [hunk 2]
```diff
@@ -1082,5 +1083,108 @@ public void VisibiltyChangedSignalEmit(bool visibility)
...
+                        {
+                            AuxiliaryMessageEventCallback = null;
+                        }
+                    }
+                }
+            }
+        }
+
+
+
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/public/Window/WindowEvent.cs:1188]: could you add the following in `DisconnectNativeSignals()`

            if (AuxiliaryMessageEventCallback != null)
            {
                using var signal = new WindowAuxiliaryMessageSignal(this);
                signal.Disconnect(AuxiliaryMessageEventCallback);
                AuxiliaryMessageEventHandler= null;
                AuxiliaryMessageEventCallback = null;
            }

thank you.

File: src/Tizen.NUI/src/public/Window/WindowEvent.cs (hunk 1 above)
Comment (sparrow74) [reply src/Tizen.NUI/src/public/Window/WindowEvent.cs:1117]: Thank you for your detail review.
I updated that.

**Review by dongsug-song — APPROVED** [review]
looks good to me

**Pull request merged**